reactioncommerce/reaction
# Default to Primary Shop when no domains match

Impact: **minor**  
Type: **feature**

## Issue
When the domain on which you are running your site does not match any Shops' `domains` property, default to the Primary Shop. (as is, you get some seeming unrelated "login" error)

## Solution
Add one more check at the end of `getShopId`, returning `getPrimaryShopId` if no shop matches.

## Testing
1. Start a fresh install of Reaction
2. open `127.0.0.1` (the Primary Shop's `domains` property is set to [`localhost`], but not the IP)
3. Open your browser's developer tools and see the error.
